errors/build-dir-not-writeable.md

```diff
@@ -0,0 +1,30 @@
+# Build directory not writeable
+
+#### Why This Error Occurred
+
+The filesystem does not allow writing to the specified directory. A common cause for this error is starting a [custom server](https://github.com/zeit/next.js#custom-server-and-routing) in development mode on a production server (for example [now.sh](https://zeit.co)).
```

Comment:
- I'd change `for example [now.sh](https://zeit.co)` to `for example, [now.sh](https://zeit.co/now), which [doesn't allow you to write to the filesystem after your app is built](https://zeit.co/docs/deployment-types/node#file-system-specifications).`